File: NBE_5_7_2_02TEAM/src/main/java/io/twogether/nbe_5_7_2_02team/oauth/dto/common/TokenBody.java
package io.twogether.nbe_5_7_2_02team.oauth.dto.common;

import io.twogether.nbe_5_7_2_02team.member.domain.Role;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
public class TokenBody {
    public Long memberId;
    public Role role;

    public TokenBody(Long memberId, Role role) {
        this.memberId = memberId;
        this.role = role;
    }
}

File: NBE_5_7_2_02TEAM/src/main/java/io/twogether/nbe_5_7_2_02team/oauth/dto/common/TokenBody.kt
package io.twogether.nbe_5_7_2_02team.oauth.dto.common

import io.swagger.v3.oas.annotations.media.Schema
import io.twogether.nbe_5_7_2_02team.member.domain.Role

@Schema(description = "토큰에 포함된 사용자 정보")
data class TokenBody(
    @Schema(description = "사용자 ID", example = "12345")
    val memberId: Long,

    @Schema(description = "사용자 권한", example = "MEMBER")
    val role: Role,
)
